### PR TITLE
Add get methods for exchange and queue

### PR DESCRIPTION
**Pull request by heckad**
https://github.com/mosquito/aio-pika/issues/281

### CODE REVIEW

**Comment by heckad** [comment]
Where to add tests?

File: aio_pika/channel.py [hunk 1]
```diff
@@ -223,13 +223,21 @@ def _on_return(self, message: aiormq.types.DeliveredMessage):
 
         return exchange
 
+    async def get_exchange(self, name: str) -> Queue:
+        """
+
```
Comment (mosquito) [inline aio_pika/channel.py]:
```
It's a shortcut for ``.declare_exchange(..., passive=True)``.
```

Comment (heckad) [reply aio_pika/channel.py]: Yes

Comment (mosquito) [reply aio_pika/channel.py]: Correct. Please add it as a docstring.

Comment (heckad) [reply aio_pika/channel.py]: I didn’t understand. Do you want copy code from 233 line and put it to 228?

Comment (mosquito) [reply aio_pika/channel.py]: Could you please add docstring for new methods.

**Comment by heckad** [comment]
@mosquito, do I need to add something else?

File: aio_pika/channel.py [hunk 2]
```diff
@@ -223,13 +223,22 @@ def _on_return(self, message: aiormq.types.DeliveredMessage):
 
         return exchange
 
+    async def get_exchange(self, name: str) -> Queue:
+        """
+        It's a shortcut for ``.declare_exchange(..., passive=True)``.
+
+        :param name: queue name
```
Comment (decaz) [inline aio_pika/channel.py]: exchange name

File: aio_pika/channel.py [hunk 3]
```diff
@@ -223,13 +223,22 @@ def _on_return(self, message: aiormq.types.DeliveredMessage):
...
+        :return: :class:`aio_pika.queue.Queue` instance
```
Comment (decaz) [inline aio_pika/channel.py]: aio_pika.exchange.Exchange

**Comment by heckad** [comment]
@mosquito ?

**Comment by heckad** [comment]
@mosquito ?

**Comment by heckad** [comment]
@mosquito, please. It's the very simple pull request. What's wrong with them?

**Comment by mosquito** [comment]
I investigated this case, and it seems that it does not work as we assume here.

if there is no entity in RabbitMQ, then get will create it. Unfortunately, this cannot be influenced.

**Comment by mosquito** [comment]
Just add test which gets an undeclared queue, and after that try to declare it with different params.
User assumes another behaviour.

**Comment by mosquito** [comment]
@decaz @heckad what we will do?

**Comment by heckad** [comment]
I asked the question how to correct to do it on https://groups.google.com/forum/#!topic/pika-python/z-vwXsm9YuQ

**Pull request merged**